### PR TITLE
[i18nIgnore] fix: images sizes and densities are not experimental

### PR DESCRIPTION
**Pull request by Princesseuh**
#### Description (required)

What the title says, those properties have never really been experimental, and are definitely not anymore these days

#### Related issues & labels (optional)

N/A

### CODE REVIEW

File: src/content/docs/en/reference/api-reference.mdx [hunk 1]
```diff
@@ -1078,7 +1078,7 @@ This function accepts the following properties:
 
 <p><Since v="2.5.0" /></p>
 
-**Type:** `'content' | 'data'`
```
Comment (liruifengv) [inline src/content/docs/en/reference/api-reference.mdx:1081]: image

These changes break the format.

File: src/content/docs/en/reference/api-reference.mdx [hunk 2]
```diff
@@ -1078,7 +1078,7 @@ This function accepts the following properties:
 
 <p><Since v="2.5.0" /></p>
 
-**Type:** `'content' | 'data'`  
+**Type:** `'content' | 'data'`
```
Comment (liruifengv) [inline src/content/docs/en/reference/api-reference.mdx:1081]: image

Hey  @Princesseuh These changes break the format. All the line breaks are broken. The original format seems intentional.

Comment (Princesseuh) [reply src/content/docs/en/reference/api-reference.mdx:1081]: Arf, I hate Markdown, auto formatting always breaks everything. I'll fix it

Comment (Princesseuh) [reply src/content/docs/en/reference/api-reference.mdx]: Fixed!

**Review by Yan-Thomas — APPROVED** [review]
oh, nice catch, thanks!

**Pull request merged**